src/components/auth/SignUpForm.js

```diff
@@ -14,17 +15,9 @@ const SignUpForm = ({ userType, onVerificationSent }) => {
   const [isExistingUser, setIsExistingUser] = useState(false);
   const [isSending, setIsSending] = useState(false);
 
-    // local storage 에서 토큰 파싱 후 로그아웃을 위한 메서드, 개발중
-    // access token이 아직 유효할 경우
-    // const handleSignInRedirect = () => {
-    //     const token = getTokenFromLocalStorage();
-    //     if (token) {
-    //         console.log("token");
-    //         navigate(`/verifyEmail?token=${encodeURIComponent(token)}`);
-    //     } else {
-    //         console.error('No token found in localStorage');
-    //     }
-    // };
+    useEffect(() => {
+        checkLoggedIn(navigate);
+    }, [navigate]);
```

Comment:
```suggestion
    }, []);
```
최초 1회 실행되는거라 의존성배열에 값이 없어도 좋을 것 같아요